### PR TITLE
Android - close buttons (chat and settings) - #1763 fix

### PR DESCRIPTION
**Pull request by ngocdh**
Fix for #1763: add close button for dialogs in Android.

### CODE REVIEW

File: src/chatdlg.cpp [hunk 1]
```diff
@@ -65,6 +65,10 @@ CChatDlg::CChatDlg ( QWidget* parent ) : CBaseDlg ( parent, Qt::Window ) // use
     connect ( action, SIGNAL ( triggered() ), this, SLOT ( close() ) );
 #endif
 
+#if defined( ANDROID ) || defined( Q_OS_ANDROID )
+    pEditMenu->addAction ( tr ( "&Close" ), this, SLOT ( close() ), QKeySequence ( Qt::CTRL + Qt::Key_C ) );
```
Comment (pljones) [inline src/chatdlg.cpp:69]: Ctrl-C opens the connections dialogue.

File: src/clientsettingsdlg.cpp [hunk 2]
```diff
@@ -42,6 +42,17 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
     layout()->setMenuBar ( pMenu );
 #endif
 
+#if defined( Q_OS_ANDROID ) || defined( ANDROID )
+    // Android too
+    QMenuBar* pMenu      = new QMenuBar ( this );
+    QMenu*    pCloseMenu = new QMenu ( tr ( "Close" ), this );
```
Comment (pljones) [inline src/clientsettingsdlg.cpp:48]: Should be `&Close` here and below?

**Review by pljones — APPROVED** [review]

**Comment by ann0see** [comment]
I don’t hab’s an Android device at hand at the moment, but can anyone test it?

File: src/clientsettingsdlg.cpp (hunk 2 above)
Comment (ann0see) [reply src/clientsettingsdlg.cpp]: Why is the & before C if we use ctrl + w?

Comment (ngocdh) [reply src/clientsettingsdlg.cpp]: Because Ctrl W is commonly known for the "Close Window" action.

Comment (ann0see) [reply src/clientsettingsdlg.cpp]: Ok. So we‘ll leave it like that?

**Comment by pljones** [comment]
Loaded up, connected, got feedback that didn't get detected, disconnected, checked in settings and feedback detection was enabled, closed settings using the menu.  Checked Chat - closed using the menu okay.

**Review by ann0see — APPROVED** [review]
Ok. So happy to approve

**Comment by ann0see** [comment]
CHANGELOG: Add close button on Android to enhance UX

**Comment by ngocdh** [comment]
Great thanks team :)

**Pull request merged**